src/OrchardCore.Modules/OrchardCore.Deployment/Steps/RecipeFileDeploymentStepDriver.cs

```diff
@@ -0,0 +1,43 @@
+using System.Threading.Tasks;
+using OrchardCore.Deployment.ViewModels;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+
+namespace OrchardCore.Deployment.Steps
+{
+    public class RecipeFileDeploymentStepDriver : DisplayDriver<DeploymentStep, RecipeFileDeploymentStep>
+    {
+        public override IDisplayResult Display(RecipeFileDeploymentStep step)
+        {
+            return
```

Comment:
Could we return lambda expression

Reply:
I don't get what you mean by 'return lambda expression' here.

Reply:
I mean lambda expression body

```csharp
public override IDisplayResult Display(RecipeFileDeploymentStep step)
    =>                 Combine(
                    View("RecipeFileDeploymentStep_Fields_Summary", step).Location("Summary", "Content"),
                    View("RecipeFileDeploymentStep_Fields_Thumbnail", step).Location("Thumbnail", "Content")
                );
```